core/src/gql/utils.rs

```diff
@@ -51,10 +52,18 @@ pub async fn get_record(
 	sess: &Session,
 	rid: &Thing,
 ) -> Result<SqlValue, GqlError> {
-	let tx = kvs.transaction(TransactionType::Read, LockType::Optimistic).await?;
-	Ok(tx
-		.get_record(sess.ns.as_ref().unwrap(), sess.db.as_ref().unwrap(), &rid.tb, &rid.id)
-		.await?
-		.as_ref()
-		.to_owned())
+	let stmt: Statement = Statement::Select(SelectStatement {
+		expr: Fields::all(),
+		what: vec![SqlValue::Thing(rid.clone())].into(),
+		only: true,
+		..Default::default()
```

Comment:
In the future we should probably support versions here.

Reply:
that would be great!